front-end-admin/src/App.vue

```diff
@@ -8,6 +8,7 @@
         <logout />
       </div>
     </navigation>
+    <br><br><br>
```

Comment:
@Tiernebre  These got added to make the navbar elements load after the navbar because for some reason they weren't. If you know of a better solution that'd be great

Reply:
Might be some CSS rule stuff for the navigation bar that is causing that issue. For now, I'm fine with using the line break elements.